skipper.env

```
@@ -128,4 +128,12 @@ INGRESS_VIP
 WORKERS_COUNT
 MASTERS_COUNT
 IPXE_BOOT
-SUBSYSTEM_LOCAL_REGISTRY
\ No newline at end of file
+SUBSYSTEM_LOCAL_REGISTRY
+NUTANIX_USERNAME
+NUTANIX_PASSWORD
+NUTANIX_ENDPOINT
```

```suggestion
NUTANIX_PRISM_ENDPOINT
```
We should clarify which endpoint is that - Prism Central or Prism Element

---

This is the same as described in Nutanix terraform documentation https://registry.terraform.io/providers/nutanix/nutanix/latest/docs#endpoint